src/sdam/description/topology/server_selection/test/logic.rs

```diff
@@ -26,62 +20,6 @@ struct TestFile {
     error: Option<bool>,
 }
 
-#[derive(Debug, Deserialize)]
```

Comment:
these were moved into `server_selection/test/mod.rs` so they could be shared between the logic and in-window tests.